test/utils/save.and.continue.spec.ts

```diff
@@ -23,15 +20,7 @@ describe('saveAndContinue test suite', () => {
   });
 
   test('should call updateOverseasEntity if feature flag is true', async () => {
```

Comment:
utils/save.and.continue.ts still has the update flag in it so we will need to keep the tests for update flag = true and update flag is false.

Reply:
Kept and added a new test